bitnami/kibana/values-production.yaml

```diff
@@ -178,7 +178,7 @@ forceInitScripts: false
 ## Service configuration
 ##
 service:
-  port: 80
+  port: 5601
```

Comment:
Do you think this change is needed? This is the service port (not the container one), for example this is the generated note (you can see it in the first deployment or using `helm status kibana)`:
```
kubectl port-forward svc/kibana 8080:80
echo "Visit http://127.0.0.1:8080 to use your application"
```

Reply:
It's needed because this is the actual port used by the container as shown [here](https://github.com/bitnami/charts/blob/d798bcaa1c965a047ffcefe8dd5c6ac00b1f5a5f/bitnami/kibana/templates/deployment.yaml#L75):
```yaml
ports:
            - name: http
              containerPort: 5601
              protocol: TCP
```
The Service works because it uses `targetPort: http` to map port 80 to 5601, but this breaks Istio routing which shows the traffic as not belonging to the mesh.

The `targetPort: http` trick is actually not needed, by changing `.Values.service.port=5601` the generated note becomes:
```
kubectl port-forward svc/kibana 8080:5601
echo "Visit http://127.0.0.1:8080 to use your application"
```